swingset/src/main/java/com/nqadmin/swingset/SSComboBox.java

```diff
@@ -76,9 +76,9 @@
  * are the correct techniques.
  * There are a variety of method to change the current combobox item
  * such as
- * {@link #setSelectedMapping(java.lang.Object) setSelectedMapping(Integer)}.
+ * {@link #setSelectedMapping(Integer)} / {@link #setSelectedItem(Object)}.
  * <p>
```

Comment:
The idea here was to point out that to change the selected item, you probably don't want to use `xxxItem()`, but rather the key as in `xxxMapping`. That should be stated more clearly.